### PR TITLE
Fixes #41 - listen to inventory's events

### PR DESCRIPTION
**Pull request by kgaikwad**
- [x] added listener to which listens `platform.inventory.events` messages
- [x] process delete event type and remove respective performance profile records from ROS DB
- [ ] refactor app.py to set app context by making use of `create-app`

### CODE REVIEW

File: ros/processors/inventory_events_consumer.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                )
+            finally:
+                self.consumer.commit()
+
+        self.consumer.close()
+
+    def host_create_update_event(self, _msg):
+        """Process create/update message."""
+        self.prefix = "PROCESSING CREATE/UPDATE EVENT"
+        LOG.info(
```
Comment (apuntamb) [inline ros/processors/inventory_events_consumer.py:76]: Is the logic for Create/Update to be added here in the same PR?

Comment (kgaikwad) [reply ros/processors/inventory_events_consumer.py:76]: https://github.com/RedHatInsights/ros-backend/pull/23#issue-569559504 
These are the only points I am going to target in this PR.

Comment (apuntamb) [reply ros/processors/inventory_events_consumer.py:76]: so should we eliminate the `host_create_update_event` part from this PR?

File: ros/processors/inventory_events_consumer.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+class InventoryEventsConsumer:
+    """Inventory events consumer."""
+
+    def __init__(self):
+        """Create a Inventory Events Consumer."""
+        self.running = True
+        self.consumer = Consumer({
+            'bootstrap.servers': INSIGHTS_KAFKA_ADDRESS,
+            'group.id': 'inventory-events-consumer',
```
[the remark this answers is not in the document]
Comment (kgaikwad) [reply ros/processors/inventory_events_consumer.py:24]: @semtexzv, 
Will it be okay to declare constant for `group-id`  & use it across the application whenever required? 
same group-id will work right?
example in [report_processor.py](https://github.com/RedHatInsights/ros-backend/blob/main/ros/processors/report_processor.py#L17)

[the remark this answers is not in the document]
Comment (semtexzv) [reply ros/processors/inventory_events_consumer.py:24]: Yeah. I'd just use 'resource-optimization'.

File: ros/processors/inventory_events_consumer.py (hunk 1 above)
Comment (kgaikwad) [reply ros/processors/inventory_events_consumer.py]: Done!

File: ros/processors/inventory_events_consumer.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        with app.app_context():
+            LOG.info(
+                'Deleting performance profile of host with insights_id %s - %s',
+                insights_id,
+                self.prefix
+            )
+            db.session.query(PerformanceProfile).filter(
+                PerformanceProfile.inventory_id == host_id
+            ).delete()
+            db.session.commit()
```
Comment (apuntamb) [inline ros/processors/inventory_events_consumer.py:85]: can we add a log saying "Performance profile of host deleted."?

**Review by apuntamb — APPROVED** [review]
LGTM! :+1:

**Comment by apuntamb** [comment]
Merged, thanks @kgaikwad !

**Pull request merged**